packages/agent/src/server.ts

```diff
@@ -35,6 +47,14 @@ export class AgentServer {
   *listen(): Operation { return; }
 
   *join(): Operation { yield; }
+
+  *launch(options: LaunchOptions): Operation<WebDriver> {
```

Comment:
this field kinda ambiguous.... maybe it should be name `launchBrowser` or `launchConnectedBrowser`